README.md

```diff
@@ -48,6 +48,7 @@ Use the "Table of Contents" menu on the top-right corner to explore the list.
 - [create-nx-workspace](https://github.com/nrwl/nx) - Scaffolding a Nx + React + Vite + Vitest.
 - [bati](https://github.com/batijs/bati) - Scaffolding a Vike project.
 - [create-awesome-node-app](https://github.com/Create-Node-App/create-node-app) - Scaffolding your project choosing between different templates.
+- [create-blastra](https://github.com/devthefuture-org/blastra/tree/main/packages/create/template) - Scaffolding a Blastra project.
```

Comment:
There should be one link, and you should specify the content of the project itself.

Reply:
I don't understand, the project itself is described here:
> [Blastra](https://github.com/devthefuture-org/blastra/) - React SSR framework built on Vite.js with file-based routing, isomorphic data loading, and optional high-performance Go server deployments.

There is one link: `[create-blastra](https://github.com/devthefuture-org/blastra/tree/main/packages/create/template)`

and the item is formatted like previous ones, eg: `[bati](https://github.com/batijs/bati) - Scaffolding a Vike project.`

Could you explain what do you expect, the requested change it's not really clear for me